src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -65,6 +65,54 @@ THE SOFTWARE.
             </div>
         </div>
 
+        <div v-if="phpServerEnabled" class="field is-horizontal">
```

Comment:
```suggestion
        <div v-if="phpServerEnabled" class="field is-horizontal is-aligned-top">
```
Looks like we lost the `is-aligned-top` class here.